src/Tickets/Commerce/Settings.php

```diff
@@ -342,6 +374,23 @@ public function get_settings() {
 
 		return array_merge( tribe( Payments_Tab::class )->get_top_level_settings(), $this->apply_commerce_enabled_conditional( $settings ) );
 	}
+	
+	/**
+	 * Returns the content for the main featured settings which displays the list of gateways.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_featured_gateways_html() {
+		$admin_views = tribe( Tribe__Tickets__Admin__Views::class );
+		$admin_views->set_template_folder( 'src/admin-views/settings/tickets-commerce/gateways' );
```

Comment:
In this case you cant change the template folder otherwise other templates that use `Tribe__Tickets__Admin__Views` would break. You either need to create a new instance of a template class. Similar to how you did on the `Featured_Settings` file.